pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v3/Transactions.java

```diff
@@ -61,6 +61,7 @@ public void getCoordinatorStats(@Suspended final AsyncResponse asyncResponse,
                                     @QueryParam("authoritative")
                                     @DefaultValue("false") boolean authoritative,
                                     @QueryParam("coordinatorId") Integer coordinatorId) {
+        checkTransactionCoordinatorEnabled();
```

Comment:
Looks like we should catch this method exception.

Reply:
It's ok, like `validateTopicName`